dbt-cta/action_builder/models/0_ctes/social_profiles_ab1.sql

```diff
@@ -12,8 +11,9 @@ select
     {{ json_extract_scalar('_airbyte_data', ['source'], ['source']) }} as source,
     {{ json_extract_scalar('_airbyte_data', ['status'], ['status']) }} as status,
     {{ json_extract_scalar('_airbyte_data', ['profile'], ['profile']) }} as profile,
-    {{ json_extract_scalar('_airbyte_data', ['entity_id'], ['entity_id']) }} as entity_id,
+    {{ json_extract_scalar('_airbyte_data', ['owner_id'], ['owner_id']) }} as owner_id,
```

Comment:
hmmm, this seems like it'll remove the `entity_id` column. Are we sure its okay to remove the column?

Reply:
Ahh, I guess I thought that there was probably a schema change where `entity_id` became `owner_id`. That said, I added `entity_id` in. If `entity_id` is empty going forward it won't be an issue, but will just be an empty column, right?

Reply:
Yep thats my understanding too! I was looking at the existing tables and it looks like this table is empty so i guess we cant really find out??

Reply:
There's less harm in keeping in jic 🤷